### PR TITLE
 Changed Chinese to English

### PR DESCRIPTION
**Pull request by Timtech4u**
Language translation from Chinese to English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,890 +1,1005 @@
...
 ### Arrows and Lexical This
 
-  「箭头」函数（`=>`）和 `this`：
+  "Arrow" function (`=>`) and `this`:
 
-  > 使用「箭头」函数我们可以体验函数式编程的”美”，高效、简洁，当然也要注意上下文 `this`。
+  > Use the "arrow" function we can experience the functional programming "beauty", efficient, simple, of course, also pay attention to the context `this`.
 
-  * e.g.
+  * E.g.
```
Comment (fundon) [inline README.md:58]: Don't need to be capitalized

File: README.md [hunk 2]
```diff
@@ -1,890 +1,1005 @@
...
-    ```js
+    `` `Js
     // old
-    var sum = function (a, b) { return a + b }
-    ```
+    Var sum = function (a, b) {return a + b}
+    `` ``
 
-    ```js
+    `` `Js
```
Comment (fundon) [inline README.md:65]: Don't need to be capitalized

File: README.md [hunk 3]
```diff
@@ -1,890 +1,1005 @@
...
 
-  * e.g.
+  * E.g.
 
-    ```js
+    `` `Js
     // old
-    function Cat () {
-      this.voice = 'miao'
+    Function cat () {
```
Comment (fundon) [inline README.md:122]: Don't need to be capitalized

File: README.md [hunk 4]
```diff
@@ -1,890 +1,1005 @@
...
+      This.voice = 'miao'
     }
     Cat.prototype.speak = function () {
-      console.log(this.voice)
+      Console.log (this.voice)
```
Comment (fundon) [inline README.md:126]: Don't need to be capitalized

File: README.md [hunk 5]
```diff
@@ -1,890 +1,1005 @@
...
 
-    ```js
+    `` `Js
     // new
-    class Cat {
-      constructor () {
-        this.voice = 'miao'
+    Class Cat {
+      Constructor () {
+        This.voice = 'miao'
```
Comment (fundon) [inline README.md:145]: Don't need to be capitalized

File: README.md [hunk 6]
```diff
@@ -1,890 +1,1005 @@
...
       }
 
-      speak () {
-        console.log(this.voice)
+      Speak () {
```
Comment (fundon) [inline README.md:148]: Don't need to be capitalized

**Comment by Timtech4u** [comment]
Thanks for the feedback @fundon , I'm closing this for a new one ;-)

**Pull request closed without merging**